### PR TITLE
chore(broker-core): clean up network configuration

### LINKED ISSUE #1928: Update configuration to address atomix changes
- only one port for internal broker communication
- one port for gateway <-> client communication
- specify a cluster name

### PR DESCRIPTION
**Pull request by deepthidevaki**
closes #1928

### CODE REVIEW

File: broker-core/src/test/java/io/zeebe/broker/system/ConfigurationTest.java [hunk 1]
```diff
@@ -72,12 +70,20 @@
...
   public static final int METRICS_PORT = MetricsCfg.DEFAULT_PORT;
   public static final int ATOMIX_PORT = SocketBindingAtomixCfg.DEFAULT_PORT;
 
+  @Test
+  public void shouldUseSpecifiedClusterName() {
+    assertClusterName("specific-cluster-name", "cluster-name");
+  }
+
+  @Test
+  public void shouldUseClusterNameromEnvironment() {
```
Comment (MiguelPires) [inline broker-core/src/test/java/io/zeebe/broker/system/ConfigurationTest.java]:
```suggestion
  public void shouldUseClusterNameFromEnvironment() {
```

**Review by miguelpires — CHANGES_REQUESTED** [review]

**Review by miguelpires — APPROVED** [review]

**Comment by deepthidevaki** [comment]
bors retry

**Pull request merged**